internal/auth0/connection/schema.go

```diff
@@ -853,3 +855,92 @@ var optionsSchema = &schema.Schema{
 		},
 	},
 }
+
+var scimResourceSchema = map[string]*schema.Schema{
+	"connection_id": {
+		Type:        schema.TypeString,
+		Required:    true,
+		ForceNew:    true,
+		Description: "ID of the connection for this SCIM configuration.",
+	},
+	"connection_name": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Name of the connection for this SCIM configuration.",
+	},
+	"strategy": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Schema of the connection for this SCIM configuration.",
+	},
+	"tenant_name": {
+		Type:        schema.TypeString,
+		Computed:    true,
+		Description: "Name of the tenant for this SCIM configuration.",
+	},
+	"user_id_attribute": {
+		Type:         schema.TypeString,
+		Optional:     true,
+		RequiredWith: []string{"user_id_attribute", "mapping"},
+		Computed:     true,
```

Comment:
Does the API set this for us in case we don't set it? Or what's making us have this computed?

Reply:
yes, the api sets it, so if it's not computed, the plan ends up being out of date